src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/domene/GrunnlagsdataDomene.kt

```diff
@@ -29,9 +29,13 @@ data class GrunnlagsdataMedMetadata(val grunnlagsdata: GrunnlagsdataDomene,
 data class GrunnlagsdataDomene(val søker: Søker,
                                val annenForelder: List<AnnenForelderMedIdent>,
                                val medlUnntak: Medlemskapsinfo,
-                               val barn: List<BarnMedIdent>
+                               val barn: List<BarnMedIdent>,
+                               val tidligereVedtaksperioder: TidligereVedtaksperioder?
```

Comment:
Default null for bakoverkompabilitet? Hvis ikke vil vel json-mapping fra DB til objekt krasje?

Reply:
Default null er kun for å initere objektet, så det går fint :) Legger til en test på dette